packages/client/src/checkout/types/checkoutOrderOperationChange.types.ts

```diff
@@ -0,0 +1,127 @@
+enum CheckoutOrderOperationChangeType {
+  ItemDiscountChanged = 'ItemDiscountChanged',
+  ItemIsUnavailable = 'ItemIsUnavailable',
+  ItemOriginChanged = 'ItemOriginChanged',
+  ItemPriceChanged = 'ItemPriceChanged',
+  ItemQuantityChanged = 'ItemQuantityChanged',
+  NotEnoughStock = 'NotEnoughStock',
+  PromoCodeEligibilityChanged = 'PromoCodeEligibilityChanged',
+  PromotionNotApplied = 'PromotionNotApplied',
+  ShippingCostChanged = 'ShippingCostChanged',
+  ShippingFlatRateEligibilityChanged = 'ShippingFlatRateEligibilityChanged',
+  TotalDiscountChanged = 'TotalDiscountChanged',
+}
+
+enum CheckoutOrderChangeReason {
+  LackOfBenefits = 'LackOfBenefits',
+  InvalidPromocode = 'InvalidPromocode',
+  NotActivePromotion = 'NotActivePromotion',
+  OnlyForFirstPurchase = 'OnlyForFirstPurchase',
+  UserNotQualifiedForPromotion = 'UserNotQualifiedForPromotion',
+  PromocodeNoLongerAvailable = 'PromocodeNoLongerAvailable',
+  PromocodeAlreadyUsedByOtherCustomer = 'PromocodeAlreadyUsedByOtherCustomer',
+  OrderNotQualifiedForPromotion = 'OrderNotQualifiedForPromotion',
+  BetterPromotionApplied = 'BetterPromotionApplied',
+  ExternalCallTimedOut = 'ExternalCallTimedOut',
+  Unknown = 'Unknownm',
```

Comment:
```suggestion
  Unknown = 'Unknown',
```

There's an extra "m" at the end of "unknown" 👀

Reply:
✅